vidlistener/listener.go

```diff
@@ -37,6 +37,8 @@ func (self *VidListener) HandleRTMPPublish(
 
 			strmID := makeStreamID(conn.URL)
 			if strmID == "" {
+				glog.Errorf("Error RTMP makeStreamID handler - empty stream ID")
```

Comment:
The error should be logged by the `makeStreamID` caller so I don't think we need this here

Reply:
Removed